server/server.lua

```diff
@@ -67,6 +67,9 @@ lib.callback.register("illenium-appearance:server:importOutfitCode", function(so
         return
     end
 
+    if playerOutfit.citizenid == citizenID then return false end -- Validation when someone tried to duplicate own outfit

```

Comment:
for both of the return statements you can just do a `return` without an explicit return parameter since it is going to be null anyways

Reply:
umm alright, wait a sec

Reply:
check again @TheiLLeniumStudios